README.md

```diff
@@ -26,4 +26,8 @@ This project uses [Node.js](nodejs.org), [npm](npmjs.com) and [gulp](http://gulp
 
 ## Security
 
-The notebook code is never made available to the client -- it is only run in our proxy server. Execution request messages from the client which contain code are ignored.
+* The notebook code is never made available to the client -- it is only run in our proxy server. Execution request messages from the client which contain code are ignored.
+* To enable HTTPS:
+    1. Generate the key and certificate files by running `make gen-certs`
```

Comment:
The Makefile generates a self-signed certificate which is fine for dev, but not OK for production. You should point this out but also be sure to say that users can bring their own real cert/key.